src/NzbDrone.Core/Download/Clients/Transmission/TransmissionBase.cs

```diff
@@ -171,12 +171,21 @@ public override void RemoveItem(DownloadClientItem item, bool deleteData)
 
         public override DownloadClientInfo GetStatus()
         {
-            var config = _proxy.GetConfig(Settings);
-            var destDir = config.DownloadDir;
+            string destDir;
 
-            if (Settings.TvCategory.IsNotNullOrWhiteSpace())
+            if (Settings.TvDirectory.IsNotNullOrWhiteSpace())
+            {
+                destDir = Settings.TvDirectory;
+            }
+            else
             {
-                destDir = string.Format("{0}/{1}", destDir, Settings.TvCategory);
+                var config = _proxy.GetConfig(Settings);
+                destDir = config.DownloadDir;
+
+                if (Settings.TvCategory.IsNotNullOrWhiteSpace())
+                {
+                    destDir = string.Format("{0}/{1}", destDir, Settings.TvCategory);
```

Comment:
Should update this to string interpolation: 
```suggestion
                    destDir = $"{destDir}/{Settings.TvCategory}";
```